apps/server/src/modules/room/api/room.controller.ts

```diff
@@ -278,4 +280,23 @@ export class RoomController {
 
 		return Promise.resolve(response);
 	}
+
+	@Post(':roomId/duplicate')
```

Comment:
Is it really a good idea to change the code naming from `copy` to `duplicate`? Thies seems not very consistent. When we decide to call the oprtation `duplicate` instead of `copy` we might have to change it in other places as well. Like e.g. in the Board.